operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -490,7 +490,6 @@ public void makeRightDomainPresence(
     DomainSpec spec = null;
     if (domain != null) {
```

Comment:
You can remove the entire if

Reply:
Is it ever possible for 'domain == null' ?

Reply:
It could be, since I think we may be handling delete cases, but you can delete the if because we're not doing anything in it.

Reply:
Removed the entire if and moved it closer to where we compare DomainSpec equality.